nautobot/core/views/generic.py

```diff
@@ -252,9 +255,25 @@ def get(self, request):
                 )
                 self.queryset = self.queryset.none()
 
+            def check_for_display(field_name, values):
```

Comment:
Shall we add one for `NautobotUIViewSet's` mixin as well? Maybe this should be a helper method in `utilities` so both places can use it without much redundant code.